src/Initializers/LogOutputBufferInitializer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tempest\Console\Initializers;
+
+use Tempest\AppConfig;
+use Tempest\Console\ConsoleConfig;
+use Tempest\Console\Output\LogOutputBuffer;
+use Tempest\Container\Container;
+use Tempest\Container\Initializer;
+use Tempest\Container\Singleton;
+use Tempest\Support\PathHelper;
+
+#[Singleton]
+final readonly class LogOutputBufferInitializer implements Initializer
+{
+    public function initialize(Container $container): LogOutputBuffer
+    {
+        $consoleConfig = $container->get(ConsoleConfig::class);
+        $appConfig = $container->get(AppConfig::class);
+
+        $path = $consoleConfig->logPath ?? PathHelper::make($appConfig->root, 'console.log');
```

Comment:
This is the place where the path to the log file is determined for background tasks.